pr-deploy/src/github.ts

```diff
@@ -21,11 +21,13 @@ type ChecksGetResponseData = Types['checks']['get']['response']['data'];
 type ChecksListForRefParams = Types['checks']['listForRef']['parameters'];
 type ChecksUpdateParams = Types['checks']['update']['parameters'];
 
-const ACTIONS: ChecksUpdateParams['actions'] = [{
-  label: 'Create a test site',
-  description: 'Serves the minified output of this PR.',
-  identifier: 'deploy-me-action',
-}];
+const ACTIONS: ChecksUpdateParams['actions'] = [
```

Comment:
While you're editing this file, it might be worth refining the GitHub status check UI. Here are screenshots of what they look like today, with some suggestions. (Feel free to punt / ignore as you see fit.)

-----------------

**Before deploy:**

<img width="812" alt="Screen Shot 2021-04-13 at 4 20 17 PM" src="https://user-images.githubusercontent.com/26553114/114616429-d9d9c880-9c74-11eb-866c-42a8a6a5db9a.png">

- Is the list of directories still correct?
- Is the time estimate of a minute still accurate? (Or are things faster with CircleCI?)

-----------------

**After deploy:**

<img width="774" alt="Screen Shot 2021-04-13 at 4 23 08 PM" src="https://user-images.githubusercontent.com/26553114/114616447-de05e600-9c74-11eb-926a-8624bb4a873a.png">

- The paragraph looks kinda crowded and hard to read. Maybe make it a bulleted list of points instead?
- Both the "website" link and the "amp article example" link point to the same page. Is there a better alternative for the "website" link?
- The "view more details" link appears to be broken (see below). Maybe this won't be a problem once the switch is complete?

<img width="1279" alt="Screen Shot 2021-04-13 at 4 23 35 PM" src="https://user-images.githubusercontent.com/26553114/114616458-e2320380-9c74-11eb-920b-31b6df1095f4.png">

Reply:
> * Is the list of directories still correct?

Updated

> * Is the time estimate of a minute still accurate? (Or are things faster with CircleCI?)

I believe on CircleCI it would be faster, but we can find out later

> * The paragraph looks kinda crowded and hard to read. Maybe make it a bulleted list of points instead?

Added line breaks in between

> * Both the "website" link and the "amp article example" link point to the same page. Is there a better alternative for the "website" link?

@estherkim?

> * The "view more details" link appears to be broken (see below). Maybe this won't be a problem once the switch is complete?

Yes, this is a result of the manual testing